.github/workflows/clair.yml

```diff
@@ -0,0 +1,37 @@
+name: Scan Theia Docker image for vulnerabilities every day
```

Comment:
If the check was skipped because of if at line 10, how can we say that the workflow has been tested? (I mean you might have tested the scheduling, but not the workflow)